variables.tf

```diff
@@ -23,3 +23,26 @@ variable "account_id" {
     "prod"    = "337779336338"
   }
 }
+
+variable "dq_pub_ips" {
+  type    = list(string)
+  default = []
+}
+
+variable "dq_ips_notprod" {
+  type = list(string)
+  default = [
+    "35.177.179.157/32",
+    "35.177.132.243/32",
+    "35.177.100.236/32"
+  ]
+}
+
+variable "dq_ips_prod" {
+  type = list(string)
+  default = [
+    "52.56.43.118/32",
```

Comment:
Are you sure about these IP addresses?
35.177.---
52.---

Reply:
these are the NAT IP addresses for the DQ AWS account - so The bastions and Tab servers will assume this IP when trying to access the the S3 endpoint